code/modules/projectiles/projectile.dm

```diff
@@ -199,8 +199,8 @@
 			M.Turn(Angle)
 			transform = M
 
-			var/Pixel_x=round(sin(Angle)+16*sin(Angle)*2)
-			var/Pixel_y=round(cos(Angle)+16*cos(Angle)*2)
+			var/Pixel_x=round((sin(Angle)+16*sin(Angle)*2), 1)	//Without the 1 it rounds to -1 for cos because byond's functions are shit.

```

Comment:
This is actually the byond round instruction being strange when given a scientifice notation number

20:31 < QualityVan> > The round instruction rounds a number to the nearest
                    specified multiple. If no rounding multiple is specified
                    (or if it is zero), the integer portion is returned. Note
                    that this is different from specifying a multiple of 1,
                    which rounds to the nearest integer.

so technically the integer portion of this value should be 0, as the number is actually 

-0.00000000000000083697

But byond is just rounding out to (at least I think so)

-1 from -1.83697e-16